addon/doxycommentview/doxycommentview.py

```diff
@@ -50,7 +58,14 @@ def do_POST(self):
                 self.send_response(200)
                 self.send_header('Content-type', 'text/html')
                 self.end_headers()
+                if VERSION:
+                    self.wfile.write(VERSION_STR.stdout.encode())
+                    self.wfile.write("<hr>".encode())
                 self.wfile.write(result.stdout.encode())
+                if DEBUG:
+                    self.wfile.write("<hr>\n<pre>".encode())
+                    self.wfile.write(result.stderr.replace('&','&amp;').replace("'","&apos;").replace('<','&lt;').replace('>','&gt;').encode())
```

Comment:
Maybe we could use [`html.encode()`](https://docs.python.org/3/library/html.html) instead

Reply:
Did you mean `html.escape` ?

I have no objections against it, I'll leave it up to you.
(The debug info helped me quite a bit for "simple" problems).